app/views/audits/show.html.erb

```diff
@@ -51,6 +51,10 @@
         <% end %>
         </tbody>
       </table>
+      <% if @audit.comment %>
+        <%= alert(:class => 'alert-info', :header => '',
```

Comment:
please include a timestamp in this view, and maybe user email address or something additionally to identify the user? I wan it to look more like a log.

(audit has a reference user object if it still exists).